docs/docs/manual/wikibase/overview.md

```diff
@@ -94,41 +96,144 @@ It will then generate a username (in the form of “yourwikidatausername@yourbot
+
+For instance, the Wikitext of [this image](https://commons.wikimedia.org/wiki/File:Harry_Whittier_Frees_-_The_Bufkins_Twins_Were_Swinging.jpg) is:
+```
+== {{int:filedesc}} ==
+{{Information
+|Description={{en|1=Illustration from "The little folks of animal land". Inscription below image: "The Bufkins Twins Were Swinging"}}
+{{ru|1=Фотография [[:w:ru:Фрис, Гарри Виттер|Гарри Виттера Фриса]] из книги «Маленький народец страны зверят» (The little folks of animal land). Подпись под картинкой: «Близнецы Бафкинс качаются» («The Bufkins Twins Were Swinging»)}}
+|Source={{cite book |author=Harry Whittier Frees |year=1915 |title=The little folks of animal land |publisher=Lothrop, Lee & Shepard co. |pages=25| url=https://books.google.com/books?id=HcwTIRt2FvwC&pg=PA25}} Retrieved from Google Books.
+|Author=[[:w:Harry Whittier Frees|Harry Whittier Frees]]
+|Date=1915
+|Permission={{PD-US}}
+|other_versions=
+}}
+
+[[Category:Photographs of kittens by Harry Whittier Frees]]
+[[Category:Cats with objects]]
+```
+
+You can extract Wikitext (and structured data statements) from a list of file names. Select <span class="menuItems">Edit column</span> → <span class="menuItems">Add columns from reconciled values...</span>
+
+See [Add columns from reconciled values](./reconciling#add-columns-from-reconciled-values) for general information about this feature.
+
+### Prepare columns with structured data
+
+Based on the Wikitext that you may have just extracted, or with data from other sources, you may have (or add) columns with additional structured data, such as the files'
+
+* [creator](https://commons.wikimedia.org/wiki/Commons:Structured_data/Modeling/Author)
+* [source](https://commons.wikimedia.org/wiki/Commons:Structured_data/Modeling/Source)
+* what is [depicted](https://commons.wikimedia.org/wiki/Commons:Structured_data/Modeling/Depiction) in the files
+* [copyright status and license](https://commons.wikimedia.org/wiki/Commons:Structured_data/Modeling/Copyright)
+
+Some of these columns will need to be reconciled with Wikidata (_not_ Wikimedia Commons!), since files on Wikimedia Commons are described with data from Wikidata.
+
+![A typical OpenRefine project with Commons files and some structured data.](/img/commons-project.png)
+
+:::info
+Data models for structured data about media files on Commons are explained and discussed at https://commons.wikimedia.org/wiki/Commons:Structured_data/Modeling.
+:::
+
+### Upload edits to Wikimedia Commons
+
+Finally, you will build a schema to model the Wikimedia Commons edits that OpenRefine will perform for each row in your project. See [schema alignment](/.schema-alignment) for general documentation about this feature.
```

Comment:
This `/.schema-alignment` link is reported by Docusaurus as being broken, and indeed it looks like it :)

Reply:
That was totally put there to just check if you were paying attention 🤪

I also added a short section about EditGroups while I was at it.

Reply:
Well I definitely did not pay attention but Docusaurus did :-D